cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -156,8 +156,6 @@ public static void after() {
                 supplier, ResourceUtils.dummyClusterOperatorConfig(1L));
         Reconciliation reconciliation = new Reconciliation("test-trigger", Kafka.RESOURCE_KIND, NAMESPACE, NAME);
 
-        Checkpoint async = context.checkpoint();
```

Comment:
Removed as unnecessary for test completion, and a lurking footgun if checkpoint ordering changed in tests using this method.